DatadogInternal/Tests/MessageBus/NewFeatureBaggageTests.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import XCTest
+@testable import DatadogInternal
+
+class NewFeatureBaggageTests: XCTestCase {
```

Comment:
We're only testing the happy path scenario in this suite.

Reply:
Indeed, I have added tests for encoding and decoding failures as well as a test for concurrency 👍